examples/schema_site_bd/main.tf

```diff
@@ -18,5 +18,6 @@ resource "mso_schema_site_bd" "foo_schema_site_bd" {
   bd_name       = "bd4"
   template_name = "Template1"
   site_id       = "5c7c95b25100008f01c1ee3c"
-  host_route          = false
+  host_route    = false
```

Comment:
Should we add a full example as we are doing with the other examples? So including data sources for tenant / site and resource for template bd?

Reply:
ok